llvm/include/llvm/ProfileData/Coverage/CoverageMapping.h

```diff
@@ -217,6 +217,9 @@ class CounterExpressionBuilder {
 
 using LineColPair = std::pair<unsigned, unsigned>;
 
+using MCDCConditionID = unsigned int;
+using MCDCConditionIDs = std::array<MCDCConditionID, 2>;
```

Comment:
is `std::array<>` better than `std::pair<>` when it only has two elements?

Reply:
`first` and `second` didn't make sense to me.